oak_functions/examples/fuzzable/module/src/tests.rs

```diff
@@ -56,7 +56,8 @@ async fn test_server() {
     let logger = Logger::for_test();
 
     let lookup_data = Arc::new(LookupData::new_empty(None, logger.clone()));
-    let lookup_factory = LookupFactory::create(lookup_data, logger.clone()).unwrap();
+    let lookup_factory =
+        LookupFactory::new_boxed_extension_factory(lookup_data, logger.clone()).unwrap();
```

Comment:
pre-existing: Why do we call `unwrap` here, but `expect` for creating `PrivateMetricsProxyFactory`? I think we want to call `expect` here too.

Reply:
done.